src/components/templates/SiteNavbar/SiteNavbar.css.ts

```diff
@@ -55,13 +56,20 @@ export const navLink = style([
 export const navActions = style({
   gridArea: 'actions',
   display: 'flex',
+  alignItems: 'center',
   gap: vars.size[3],
 });
 
 export const navAction = style([
   navItemBase,
   {
     borderRadius: vars.borderRadius.lg,
-    height: vars.size[8],
+    height: '32px', // temp fix because some icons had extra height for an unknown reason
```

Comment:
Can you start comments like this with `TODO: ` so they're easy to search for?

Reply:
fixed